### PR TITLE
NATIVE-470: Add alt text for header icons

### PR DESCRIPTION
**Pull request by NKlug**

### CODE REVIEW

File: src/modules/layout/components/Header.js [hunk 1]
```diff
@@ -160,14 +160,15 @@ class Header extends React.PureComponent<PropsType> {
     return <BoxShadow theme={theme}>
       <Horizontal>
         <HorizontalLeft>
-          {this.canGoBackInStack() ? <HeaderBackButton onPress={this.goBackInStack} /> : <Logo source={logo} />}
+          {this.canGoBackInStack() ? <HeaderBackButton onPress={this.goBackInStack} />
+          : <Logo source={logo} accessibilityLabel={'Integreat'} />}
```
Comment (klinzo) [inline src/modules/layout/components/Header.js]: Maybe `Home` or `Dashboard` would be better here.

File: src/modules/layout/components/Header.js [hunk 2]
```diff
@@ -144,12 +144,12 @@ class Header extends React.PureComponent<PropsType> {
 
   renderItem (
     title: string, iconName?: string, show: 'never' | 'always',
-    onPress: ?() => void | Promise<void>
+    onPress: ?() => void | Promise<void>, accessibilityLabel: ?string = undefined
```
Comment (maxammann) [inline src/modules/layout/components/Header.js]: Maybe we should not make this optional. Such that every icon gets an alt text.

File: src/modules/layout/components/Header.js (hunk 1 above)
Comment (klinzo) [reply src/modules/layout/components/Header.js]: @maxammann @Schedulaar what do you think?

Comment (maxammann) [reply src/modules/layout/components/Header.js]: Does this always go to the dashboard?

Comment (klinzo) [reply src/modules/layout/components/Header.js]: I tink so, yes

Comment (Schedulaar) [reply src/modules/layout/components/Header.js]: No it's only display when we're already on the Dashboard. So either we use an empty alt text or 'Integreat Logo'. (But probably rather empty)

Comment (maxammann) [reply src/modules/layout/components/Header.js]: Rather empty those thumb ups are for :)

**Comment by maxammann** [comment]
Merged in develop

**Pull request merged**